core/modules/replica/Controller.h

```diff
@@ -103,204 +256,304 @@ class Controller : public std::enable_shared_from_this<Controller> {
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    SqlDisableDbRequestPtr sqlDisableDb(
+            std::string const& workerName,
+            std::string const& database,
+            std::function<void(SqlDisableDbRequestPtr)> const& onFinish=nullptr,
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    SqlGrantAccessRequestPtr sqlGrantAccess(
+            std::string const& workerName,
+            std::string const& database,
+            std::string const& user,
+            std::function<void(SqlGrantAccessRequestPtr)> const& onFinish=nullptr,
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    SqlCreateTableRequestPtr sqlCreateTable(
+            std::string const& workerName,
+            std::string const& database,
+            std::string const& table,
+            std::string const& engine,
+            std::string const& partitionByColumn,
+            std::list<std::pair<std::string, std::string>> const& columns,
+            std::function<void(SqlCreateTableRequestPtr)> const& onFinish=nullptr,
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    SqlDeleteTableRequestPtr sqlDeleteTable(
+            std::string const& workerName,
+            std::string const& database,
+            std::string const& table,
+            std::function<void(SqlDeleteTableRequestPtr)> const& onFinish=nullptr,
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    SqlRemoveTablePartitionsRequestPtr sqlRemoveTablePartitions(
+            std::string const& workerName,
+            std::string const& database,
+            std::string const& table,
+            std::function<void(SqlRemoveTablePartitionsRequestPtr)> const& onFinish=nullptr,
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    SqlDeleteTablePartitionRequestPtr sqlDeleteTablePartition(
+            std::string const& workerName,
+            std::string const& database,
+            std::string const& table,
+            uint32_t transactionId,
+            std::function<void(SqlDeleteTablePartitionRequestPtr)> const& onFinish=nullptr,
+            int priority=0,
+            bool keepTracking=true,
+            std::string const& jobId="",
+            unsigned int requestExpirationIvalSec=0);
+
+    template <class REQUEST>
+    typename REQUEST::Ptr stopById(
```

Comment:
is it worth documenting any of these?

Reply:
Well, after receiving cute a bit of criticism for the Doxygen-style comments :smh I started culling the comments in favor of the "self-documented" code. In this particular case the method is used to stop a request of a specific type. Class `Controller` does 3 things:
* starts new requests
* stops requests by their identifiers
* reports a status of the ongoing requests
These three roles of the class are explained in the comment section above class definition. And I tough, the rest should be more-or-less obvious.